spec/rails_helper.rb

```diff
@@ -60,4 +60,8 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
```

Comment:
maybe worth mentioning explicitly the test helper method `sign_in(resource)` which these mixins provide? A link to the docs at https://github.com/heartcombo/devise?tab=readme-ov-file#test-helpers might be useful too?